[2024-12-27T16:35:14.806+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-27T16:35:17.745+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: movie_pipeline.scrape_filmmakers manual__2024-12-27T11:38:32.797674+00:00 [queued]>
[2024-12-27T16:35:17.877+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: movie_pipeline.scrape_filmmakers manual__2024-12-27T11:38:32.797674+00:00 [queued]>
[2024-12-27T16:35:17.891+0000] {taskinstance.py:2866} INFO - Starting attempt 2 of 2
[2024-12-27T16:35:17.986+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): scrape_filmmakers> on 2024-12-27 11:38:32.797674+00:00
[2024-12-27T16:35:18.056+0000] {standard_task_runner.py:72} INFO - Started process 267 to run task
[2024-12-27T16:35:18.091+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'movie_pipeline', 'scrape_filmmakers', 'manual__2024-12-27T11:38:32.797674+00:00', '--job-id', '78', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_dag.py', '--cfg-path', '/tmp/tmpqva4lkds']
[2024-12-27T16:35:18.123+0000] {standard_task_runner.py:105} INFO - Job 78: Subtask scrape_filmmakers
[2024-12-27T16:35:19.078+0000] {task_command.py:467} INFO - Running <TaskInstance: movie_pipeline.scrape_filmmakers manual__2024-12-27T11:38:32.797674+00:00 [running]> on host a437d9ca329c
[2024-12-27T16:35:19.985+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='movie_pipeline' AIRFLOW_CTX_TASK_ID='scrape_filmmakers' AIRFLOW_CTX_EXECUTION_DATE='2024-12-27T11:38:32.797674+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-27T11:38:32.797674+00:00'
[2024-12-27T16:35:19.991+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-27T16:35:42.517+0000] {extract.py:151} INFO - movie iteration 0 and filmamker iteration 0 for movie_id : tt13186482 and filmmaker_id: nm1503575
[2024-12-27T16:36:22.348+0000] {extract.py:151} INFO - movie iteration 0 and filmamker iteration 1 for movie_id : tt13186482 and filmmaker_id: nm0622288
[2024-12-27T16:37:06.330+0000] {extract.py:151} INFO - movie iteration 0 and filmamker iteration 2 for movie_id : tt13186482 and filmmaker_id: nm0941314
[2024-12-27T16:37:47.872+0000] {extract.py:151} INFO - movie iteration 0 and filmamker iteration 3 for movie_id : tt13186482 and filmmaker_id: nm0575293
[2024-12-27T16:38:19.664+0000] {extract.py:151} INFO - movie iteration 0 and filmamker iteration 4 for movie_id : tt13186482 and filmmaker_id: nm7543974
[2024-12-27T16:38:47.048+0000] {extract.py:151} INFO - movie iteration 0 and filmamker iteration 5 for movie_id : tt13186482 and filmmaker_id: nm5123156
[2024-12-27T16:39:20.092+0000] {extract.py:151} INFO - movie iteration 0 and filmamker iteration 6 for movie_id : tt13186482 and filmmaker_id: nm2274707
[2024-12-27T16:39:54.934+0000] {extract.py:151} INFO - movie iteration 1 and filmamker iteration 0 for movie_id : tt5040012 and filmmaker_id: nm3211470
[2024-12-27T16:40:17.527+0000] {extract.py:151} INFO - movie iteration 1 and filmamker iteration 1 for movie_id : tt5040012 and filmmaker_id: nm3211470
[2024-12-27T16:40:51.729+0000] {extract.py:151} INFO - movie iteration 1 and filmamker iteration 2 for movie_id : tt5040012 and filmmaker_id: nm0301961
[2024-12-27T16:41:16.818+0000] {extract.py:151} INFO - movie iteration 1 and filmamker iteration 3 for movie_id : tt5040012 and filmmaker_id: nm0831290
[2024-12-27T16:41:48.779+0000] {extract.py:151} INFO - movie iteration 1 and filmamker iteration 4 for movie_id : tt5040012 and filmmaker_id: nm6675440
[2024-12-27T16:42:23.666+0000] {extract.py:151} INFO - movie iteration 1 and filmamker iteration 5 for movie_id : tt5040012 and filmmaker_id: nm0396558
[2024-12-27T16:43:00.021+0000] {extract.py:151} INFO - movie iteration 1 and filmamker iteration 6 for movie_id : tt5040012 and filmmaker_id: nm0803889
[2024-12-27T16:43:34.394+0000] {extract.py:151} INFO - movie iteration 2 and filmamker iteration 0 for movie_id : tt18259086 and filmmaker_id: nm1733778
[2024-12-27T16:44:08.938+0000] {extract.py:151} INFO - movie iteration 2 and filmamker iteration 1 for movie_id : tt18259086 and filmmaker_id: nm1352459
[2024-12-27T16:44:30.574+0000] {extract.py:151} INFO - movie iteration 2 and filmamker iteration 2 for movie_id : tt18259086 and filmmaker_id: nm1351254
[2024-12-27T16:44:56.803+0000] {extract.py:151} INFO - movie iteration 2 and filmamker iteration 3 for movie_id : tt18259086 and filmmaker_id: nm4465847
[2024-12-27T16:45:22.256+0000] {extract.py:151} INFO - movie iteration 2 and filmamker iteration 4 for movie_id : tt18259086 and filmmaker_id: nm0000120
[2024-12-27T16:45:52.484+0000] {extract.py:151} INFO - movie iteration 2 and filmamker iteration 5 for movie_id : tt18259086 and filmmaker_id: nm2355635
[2024-12-27T16:46:31.555+0000] {extract.py:151} INFO - movie iteration 2 and filmamker iteration 6 for movie_id : tt18259086 and filmmaker_id: nm0000206
[2024-12-27T16:47:06.587+0000] {extract.py:151} INFO - movie iteration 3 and filmamker iteration 0 for movie_id : tt11563598 and filmmaker_id: nm0003506
[2024-12-27T16:47:41.168+0000] {extract.py:151} INFO - movie iteration 3 and filmamker iteration 1 for movie_id : tt11563598 and filmmaker_id: nm0003506
[2024-12-27T16:48:15.186+0000] {extract.py:151} INFO - movie iteration 3 and filmamker iteration 2 for movie_id : tt11563598 and filmmaker_id: nm0168379
[2024-12-27T16:48:39.580+0000] {extract.py:151} INFO - movie iteration 3 and filmamker iteration 3 for movie_id : tt11563598 and filmmaker_id: nm2374778
[2024-12-27T16:48:58.315+0000] {extract.py:151} INFO - movie iteration 3 and filmamker iteration 4 for movie_id : tt11563598 and filmmaker_id: nm3154303
[2024-12-27T16:49:23.287+0000] {extract.py:151} INFO - movie iteration 3 and filmamker iteration 5 for movie_id : tt11563598 and filmmaker_id: nm0001570
[2024-12-27T16:49:49.610+0000] {extract.py:151} INFO - movie iteration 3 and filmamker iteration 6 for movie_id : tt11563598 and filmmaker_id: nm1102577
[2024-12-27T16:50:17.420+0000] {extract.py:151} INFO - movie iteration 4 and filmamker iteration 0 for movie_id : tt1262426 and filmmaker_id: nm0160840
[2024-12-27T16:50:54.857+0000] {extract.py:151} INFO - movie iteration 4 and filmamker iteration 1 for movie_id : tt1262426 and filmmaker_id: nm0392848
[2024-12-27T16:51:19.129+0000] {extract.py:151} INFO - movie iteration 4 and filmamker iteration 2 for movie_id : tt1262426 and filmmaker_id: nm1401416
[2024-12-27T16:51:43.925+0000] {extract.py:151} INFO - movie iteration 4 and filmamker iteration 3 for movie_id : tt1262426 and filmmaker_id: nm0536579
[2024-12-27T16:52:02.777+0000] {extract.py:151} INFO - movie iteration 4 and filmamker iteration 4 for movie_id : tt1262426 and filmmaker_id: nm7248827
[2024-12-27T16:52:26.420+0000] {extract.py:151} INFO - movie iteration 4 and filmamker iteration 5 for movie_id : tt1262426 and filmmaker_id: nm3812858
[2024-12-27T16:52:47.436+0000] {extract.py:151} INFO - movie iteration 4 and filmamker iteration 6 for movie_id : tt1262426 and filmmaker_id: nm0000156
[2024-12-27T16:53:18.486+0000] {extract.py:151} INFO - movie iteration 5 and filmamker iteration 0 for movie_id : tt13622970 and filmmaker_id: nm3989471
[2024-12-27T16:53:53.559+0000] {extract.py:151} INFO - movie iteration 5 and filmamker iteration 1 for movie_id : tt13622970 and filmmaker_id: nm2911364
[2024-12-27T16:54:22.980+0000] {extract.py:151} INFO - movie iteration 5 and filmamker iteration 2 for movie_id : tt13622970 and filmmaker_id: nm2834849
[2024-12-27T16:54:57.361+0000] {extract.py:151} INFO - movie iteration 5 and filmamker iteration 3 for movie_id : tt13622970 and filmmaker_id: nm1158544
[2024-12-27T16:55:27.631+0000] {extract.py:151} INFO - movie iteration 5 and filmamker iteration 4 for movie_id : tt13622970 and filmmaker_id: nm2834849
[2024-12-27T16:55:56.427+0000] {extract.py:151} INFO - movie iteration 5 and filmamker iteration 5 for movie_id : tt13622970 and filmmaker_id: nm2414279
[2024-12-27T16:56:19.634+0000] {extract.py:151} INFO - movie iteration 5 and filmamker iteration 6 for movie_id : tt13622970 and filmmaker_id: nm7635388
[2024-12-27T16:56:45.352+0000] {extract.py:151} INFO - movie iteration 5 and filmamker iteration 7 for movie_id : tt13622970 and filmmaker_id: nm0425005
[2024-12-27T16:57:19.266+0000] {extract.py:151} INFO - movie iteration 5 and filmamker iteration 8 for movie_id : tt13622970 and filmmaker_id: nm12533314
[2024-12-27T16:57:42.984+0000] {extract.py:151} INFO - movie iteration 6 and filmamker iteration 0 for movie_id : tt6133444 and filmmaker_id: nm1121126
[2024-12-27T16:58:11.752+0000] {extract.py:151} INFO - movie iteration 6 and filmamker iteration 1 for movie_id : tt6133444 and filmmaker_id: nm1503575
[2024-12-27T16:58:37.241+0000] {extract.py:151} INFO - movie iteration 6 and filmamker iteration 2 for movie_id : tt6133444 and filmmaker_id: nm3629113
[2024-12-27T16:59:06.162+0000] {extract.py:151} INFO - movie iteration 6 and filmamker iteration 3 for movie_id : tt6133444 and filmmaker_id: nm3109964
[2024-12-27T16:59:38.155+0000] {extract.py:151} INFO - movie iteration 6 and filmamker iteration 4 for movie_id : tt6133444 and filmmaker_id: nm12380497
[2024-12-27T17:00:03.219+0000] {extract.py:151} INFO - movie iteration 7 and filmamker iteration 0 for movie_id : tt30057084 and filmmaker_id: nm0717603
[2024-12-27T17:00:40.428+0000] {extract.py:151} INFO - movie iteration 7 and filmamker iteration 1 for movie_id : tt30057084 and filmmaker_id: nm0717603
[2024-12-27T17:01:21.661+0000] {extract.py:151} INFO - movie iteration 7 and filmamker iteration 2 for movie_id : tt30057084 and filmmaker_id: nm0000173
[2024-12-27T17:02:01.095+0000] {job.py:229} INFO - Heartbeat recovered after 31.18 seconds
[2024-12-27T17:02:31.204+0000] {extract.py:151} INFO - movie iteration 7 and filmamker iteration 3 for movie_id : tt30057084 and filmmaker_id: nm6170168
[2024-12-27T17:02:48.655+0000] {extract.py:151} INFO - movie iteration 7 and filmamker iteration 4 for movie_id : tt30057084 and filmmaker_id: nm0000104
[2024-12-27T17:03:12.182+0000] {extract.py:151} INFO - movie iteration 8 and filmamker iteration 0 for movie_id : tt9218128 and filmmaker_id: nm0000631
[2024-12-27T17:03:38.485+0000] {extract.py:151} INFO - movie iteration 8 and filmamker iteration 1 for movie_id : tt9218128 and filmmaker_id: nm0291905
[2024-12-27T17:03:56.355+0000] {extract.py:151} INFO - movie iteration 8 and filmamker iteration 2 for movie_id : tt9218128 and filmmaker_id: nm0185976
[2024-12-27T17:04:16.612+0000] {extract.py:151} INFO - movie iteration 8 and filmamker iteration 3 for movie_id : tt9218128 and filmmaker_id: nm0769227
[2024-12-27T17:04:35.745+0000] {extract.py:151} INFO - movie iteration 8 and filmamker iteration 4 for movie_id : tt9218128 and filmmaker_id: nm8958770
[2024-12-27T17:04:53.677+0000] {extract.py:151} INFO - movie iteration 8 and filmamker iteration 5 for movie_id : tt9218128 and filmmaker_id: nm0000243
[2024-12-27T17:05:16.780+0000] {extract.py:151} INFO - movie iteration 8 and filmamker iteration 6 for movie_id : tt9218128 and filmmaker_id: nm0050959
[2024-12-27T17:05:35.878+0000] {extract.py:151} INFO - movie iteration 9 and filmamker iteration 0 for movie_id : tt29137778 and filmmaker_id: nm1351912
[2024-12-27T17:05:53.944+0000] {extract.py:151} INFO - movie iteration 9 and filmamker iteration 1 for movie_id : tt29137778 and filmmaker_id: nm3220108
[2024-12-27T17:06:19.546+0000] {extract.py:151} INFO - movie iteration 9 and filmamker iteration 2 for movie_id : tt29137778 and filmmaker_id: nm4137576
[2024-12-27T17:06:37.302+0000] {extract.py:151} INFO - movie iteration 9 and filmamker iteration 3 for movie_id : tt29137778 and filmmaker_id: nm7918285
[2024-12-27T17:06:51.245+0000] {extract.py:151} INFO - movie iteration 9 and filmamker iteration 4 for movie_id : tt29137778 and filmmaker_id: nm1685408
[2024-12-27T17:07:09.056+0000] {extract.py:151} INFO - movie iteration 9 and filmamker iteration 5 for movie_id : tt29137778 and filmmaker_id: nm0568180
[2024-12-27T17:07:31.114+0000] {extract.py:151} INFO - movie iteration 9 and filmamker iteration 6 for movie_id : tt29137778 and filmmaker_id: nm0592874
[2024-12-27T17:07:48.525+0000] {extract.py:151} INFO - movie iteration 10 and filmamker iteration 0 for movie_id : tt8790086 and filmmaker_id: nm1170855
[2024-12-27T17:08:07.150+0000] {extract.py:151} INFO - movie iteration 10 and filmamker iteration 1 for movie_id : tt8790086 and filmmaker_id: nm0921013
[2024-12-27T17:08:45.473+0000] {extract.py:151} INFO - movie iteration 10 and filmamker iteration 2 for movie_id : tt8790086 and filmmaker_id: nm1436466
[2024-12-27T17:09:02.538+0000] {extract.py:151} INFO - movie iteration 10 and filmamker iteration 3 for movie_id : tt8790086 and filmmaker_id: nm0391344
[2024-12-27T17:09:21.621+0000] {extract.py:151} INFO - movie iteration 10 and filmamker iteration 4 for movie_id : tt8790086 and filmmaker_id: nm1093951
[2024-12-27T17:09:47.905+0000] {extract.py:151} INFO - movie iteration 10 and filmamker iteration 5 for movie_id : tt8790086 and filmmaker_id: nm3663196
[2024-12-27T17:10:09.868+0000] {extract.py:151} INFO - movie iteration 10 and filmamker iteration 6 for movie_id : tt8790086 and filmmaker_id: nm9538520
[2024-12-27T17:10:26.082+0000] {extract.py:151} INFO - movie iteration 11 and filmamker iteration 0 for movie_id : tt14948432 and filmmaker_id: nm0440458
[2024-12-27T17:10:48.647+0000] {extract.py:151} INFO - movie iteration 11 and filmamker iteration 1 for movie_id : tt14948432 and filmmaker_id: nm0604555
[2024-12-27T17:11:09.847+0000] {extract.py:151} INFO - movie iteration 11 and filmamker iteration 2 for movie_id : tt14948432 and filmmaker_id: nm1205652
[2024-12-27T17:11:44.039+0000] {extract.py:151} INFO - movie iteration 11 and filmamker iteration 3 for movie_id : tt14948432 and filmmaker_id: nm0425005
[2024-12-27T17:12:05.891+0000] {extract.py:151} INFO - movie iteration 11 and filmamker iteration 4 for movie_id : tt14948432 and filmmaker_id: nm0262635
[2024-12-27T17:12:27.288+0000] {extract.py:151} INFO - movie iteration 11 and filmamker iteration 5 for movie_id : tt14948432 and filmmaker_id: nm0005154
[2024-12-27T17:12:45.151+0000] {extract.py:151} INFO - movie iteration 12 and filmamker iteration 0 for movie_id : tt2347285 and filmmaker_id: nm0420817
[2024-12-27T17:13:00.501+0000] {extract.py:151} INFO - movie iteration 12 and filmamker iteration 1 for movie_id : tt2347285 and filmmaker_id: nm0732393
[2024-12-27T17:13:12.810+0000] {extract.py:151} INFO - movie iteration 12 and filmamker iteration 2 for movie_id : tt2347285 and filmmaker_id: nm1480232
[2024-12-27T17:13:38.124+0000] {extract.py:151} INFO - movie iteration 12 and filmamker iteration 3 for movie_id : tt2347285 and filmmaker_id: nm3056833
[2024-12-27T17:13:50.620+0000] {extract.py:151} INFO - movie iteration 12 and filmamker iteration 4 for movie_id : tt2347285 and filmmaker_id: nm14398475
[2024-12-27T17:14:07.961+0000] {extract.py:151} INFO - movie iteration 12 and filmamker iteration 5 for movie_id : tt2347285 and filmmaker_id: nm8407821
[2024-12-27T17:14:21.439+0000] {extract.py:151} INFO - movie iteration 12 and filmamker iteration 6 for movie_id : tt2347285 and filmmaker_id: nm0339460
[2024-12-27T17:14:44.237+0000] {extract.py:151} INFO - movie iteration 13 and filmamker iteration 0 for movie_id : tt14824600 and filmmaker_id: nm0436784
[2024-12-27T17:15:03.463+0000] {extract.py:151} INFO - movie iteration 13 and filmamker iteration 1 for movie_id : tt14824600 and filmmaker_id: nm2731620
[2024-12-27T17:15:18.498+0000] {extract.py:151} INFO - movie iteration 13 and filmamker iteration 2 for movie_id : tt14824600 and filmmaker_id: nm2176993
[2024-12-27T17:15:40.390+0000] {extract.py:151} INFO - movie iteration 13 and filmamker iteration 3 for movie_id : tt14824600 and filmmaker_id: nm3907831
[2024-12-27T17:15:54.821+0000] {extract.py:151} INFO - movie iteration 13 and filmamker iteration 4 for movie_id : tt14824600 and filmmaker_id: nm0004051
[2024-12-27T17:16:14.307+0000] {extract.py:151} INFO - movie iteration 13 and filmamker iteration 5 for movie_id : tt14824600 and filmmaker_id: nm3230329
[2024-12-27T17:16:28.640+0000] {extract.py:151} INFO - movie iteration 13 and filmamker iteration 6 for movie_id : tt14824600 and filmmaker_id: nm0001584
[2024-12-27T17:16:46.699+0000] {extract.py:151} INFO - movie iteration 14 and filmamker iteration 0 for movie_id : tt8999762 and filmmaker_id: nm1227232
[2024-12-27T17:17:03.454+0000] {extract.py:151} INFO - movie iteration 14 and filmamker iteration 1 for movie_id : tt8999762 and filmmaker_id: nm1227232
[2024-12-27T17:17:16.850+0000] {extract.py:151} INFO - movie iteration 14 and filmamker iteration 2 for movie_id : tt8999762 and filmmaker_id: nm2178754
[2024-12-27T17:17:29.666+0000] {extract.py:151} INFO - movie iteration 14 and filmamker iteration 3 for movie_id : tt8999762 and filmmaker_id: nm0004778
[2024-12-27T17:17:49.606+0000] {extract.py:151} INFO - movie iteration 14 and filmamker iteration 4 for movie_id : tt8999762 and filmmaker_id: nm0428065
[2024-12-27T17:18:14.041+0000] {extract.py:151} INFO - movie iteration 14 and filmamker iteration 5 for movie_id : tt8999762 and filmmaker_id: nm0001602
[2024-12-27T17:18:32.783+0000] {extract.py:151} INFO - movie iteration 15 and filmamker iteration 0 for movie_id : tt21823606 and filmmaker_id: nm0251986
[2024-12-27T17:18:53.604+0000] {extract.py:151} INFO - movie iteration 15 and filmamker iteration 1 for movie_id : tt21823606 and filmmaker_id: nm0251986
[2024-12-27T17:19:24.557+0000] {extract.py:151} INFO - movie iteration 15 and filmamker iteration 2 for movie_id : tt21823606 and filmmaker_id: nm0001085
[2024-12-27T17:19:45.361+0000] {extract.py:151} INFO - movie iteration 15 and filmamker iteration 3 for movie_id : tt21823606 and filmmaker_id: nm0251986
[2024-12-27T17:20:03.668+0000] {extract.py:151} INFO - movie iteration 15 and filmamker iteration 4 for movie_id : tt21823606 and filmmaker_id: nm6159473
[2024-12-27T17:20:18.037+0000] {extract.py:151} INFO - movie iteration 16 and filmamker iteration 0 for movie_id : tt20215234 and filmmaker_id: nm0074163
[2024-12-27T17:20:34.103+0000] {extract.py:151} INFO - movie iteration 16 and filmamker iteration 1 for movie_id : tt20215234 and filmmaker_id: nm1661186
[2024-12-27T17:20:49.643+0000] {extract.py:151} INFO - movie iteration 16 and filmamker iteration 2 for movie_id : tt20215234 and filmmaker_id: nm0365249
[2024-12-27T17:21:05.002+0000] {extract.py:151} INFO - movie iteration 16 and filmamker iteration 3 for movie_id : tt20215234 and filmmaker_id: nm0000146
[2024-12-27T17:21:25.959+0000] {extract.py:151} INFO - movie iteration 16 and filmamker iteration 4 for movie_id : tt20215234 and filmmaker_id: nm0001804
[2024-12-27T17:21:48.972+0000] {extract.py:151} INFO - movie iteration 16 and filmamker iteration 5 for movie_id : tt20215234 and filmmaker_id: nm0001475
[2024-12-27T17:22:11.926+0000] {extract.py:151} INFO - movie iteration 17 and filmamker iteration 0 for movie_id : tt23055660 and filmmaker_id: nm8840476
[2024-12-27T17:22:25.517+0000] {extract.py:151} INFO - movie iteration 17 and filmamker iteration 1 for movie_id : tt23055660 and filmmaker_id: nm8840476
[2024-12-27T17:22:38.586+0000] {extract.py:151} INFO - movie iteration 17 and filmamker iteration 2 for movie_id : tt23055660 and filmmaker_id: nm0055656
[2024-12-27T17:23:02.335+0000] {extract.py:151} INFO - movie iteration 17 and filmamker iteration 3 for movie_id : tt23055660 and filmmaker_id: nm1019209
[2024-12-27T17:23:20.930+0000] {extract.py:151} INFO - movie iteration 17 and filmamker iteration 4 for movie_id : tt23055660 and filmmaker_id: nm5376232
[2024-12-27T17:23:39.592+0000] {extract.py:151} INFO - movie iteration 17 and filmamker iteration 5 for movie_id : tt23055660 and filmmaker_id: nm1518261
[2024-12-27T17:24:03.329+0000] {extract.py:151} INFO - movie iteration 17 and filmamker iteration 6 for movie_id : tt23055660 and filmmaker_id: nm0254712
[2024-12-27T17:24:32.170+0000] {extract.py:151} INFO - movie iteration 18 and filmamker iteration 0 for movie_id : tt14260836 and filmmaker_id: nm1243905
[2024-12-27T17:25:16.504+0000] {extract.py:151} INFO - movie iteration 18 and filmamker iteration 1 for movie_id : tt14260836 and filmmaker_id: nm2352568
[2024-12-27T17:25:50.836+0000] {extract.py:151} INFO - movie iteration 18 and filmamker iteration 2 for movie_id : tt14260836 and filmmaker_id: nm2555314
[2024-12-27T17:26:17.901+0000] {extract.py:151} INFO - movie iteration 18 and filmamker iteration 3 for movie_id : tt14260836 and filmmaker_id: nm1243905
[2024-12-27T17:26:46.043+0000] {extract.py:151} INFO - movie iteration 18 and filmamker iteration 4 for movie_id : tt14260836 and filmmaker_id: nm0005560
[2024-12-27T17:27:32.536+0000] {extract.py:151} INFO - movie iteration 18 and filmamker iteration 5 for movie_id : tt14260836 and filmmaker_id: nm5956650
[2024-12-27T17:28:07.901+0000] {extract.py:151} INFO - movie iteration 18 and filmamker iteration 6 for movie_id : tt14260836 and filmmaker_id: nm0671487
[2024-12-27T17:28:37.371+0000] {extract.py:151} INFO - movie iteration 19 and filmamker iteration 0 for movie_id : tt26446278 and filmmaker_id: nm0478799
[2024-12-27T17:29:07.237+0000] {extract.py:151} INFO - movie iteration 19 and filmamker iteration 1 for movie_id : tt26446278 and filmmaker_id: nm1141181
[2024-12-27T17:29:42.884+0000] {extract.py:151} INFO - movie iteration 19 and filmamker iteration 2 for movie_id : tt26446278 and filmmaker_id: nm0241416
[2024-12-27T17:30:22.767+0000] {extract.py:151} INFO - movie iteration 19 and filmamker iteration 3 for movie_id : tt26446278 and filmmaker_id: nm0478799
[2024-12-27T17:31:57.328+0000] {job.py:229} INFO - Heartbeat recovered after 13.45 seconds
[2024-12-27T17:32:47.598+0000] {job.py:229} INFO - Heartbeat recovered after 51.09 seconds
[2024-12-27T17:33:00.242+0000] {extract.py:151} INFO - movie iteration 19 and filmamker iteration 4 for movie_id : tt26446278 and filmmaker_id: nm1141181
[2024-12-27T17:35:15.453+0000] {connectionpool.py:827} WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPConnectionPool(host='localhost', port=59133): Read timed out. (read timeout=120)")': /session/9253bf0a7ea50665da88900a863fd7b2/source
[2024-12-27T17:38:39.021+0000] {connectionpool.py:827} WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPConnectionPool(host='localhost', port=59133): Read timed out. (read timeout=120)")': /session/9253bf0a7ea50665da88900a863fd7b2/source
[2024-12-27T17:40:23.951+0000] {job.py:229} INFO - Heartbeat recovered after 428.70 seconds
[2024-12-27T17:40:40.301+0000] {local_task_job_runner.py:346} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2024-12-27T17:40:40.347+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2024-12-27T17:40:40.392+0000] {process_utils.py:132} INFO - Sending 15 to group 267. PIDs of all processes in the group: [427, 433, 437, 438, 447, 449, 476, 740, 451, 477, 510, 8700, 8731, 475, 267]
[2024-12-27T17:40:40.400+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 267
[2024-12-27T17:40:40.526+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=8731, status='terminated', started='17:40:36') (8731) terminated with exit code None
[2024-12-27T17:40:40.531+0000] {taskinstance.py:3093} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-12-27T17:40:40.607+0000] {connectionpool.py:827} WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fb42c9be670>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/9253bf0a7ea50665da88900a863fd7b2/window
[2024-12-27T17:40:40.613+0000] {connectionpool.py:827} WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fb42c9be280>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/9253bf0a7ea50665da88900a863fd7b2/window
[2024-12-27T17:40:40.622+0000] {connectionpool.py:827} WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fb42c9be940>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/9253bf0a7ea50665da88900a863fd7b2/window
[2024-12-27T17:40:40.715+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=447, status='terminated', started='16:35:38') (447) terminated with exit code None
[2024-12-27T17:40:40.726+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=451, status='terminated', started='16:35:38') (451) terminated with exit code None
[2024-12-27T17:40:40.738+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=438, status='terminated', started='16:35:38') (438) terminated with exit code None
[2024-12-27T17:40:40.740+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=433, status='terminated', started='16:35:38') (433) terminated with exit code None
[2024-12-27T17:40:40.746+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=740, status='terminated', started='16:37:15') (740) terminated with exit code None
[2024-12-27T17:40:40.749+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=477, status='terminated', started='16:35:39') (477) terminated with exit code None
[2024-12-27T17:40:40.781+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=437, status='terminated', started='16:35:38') (437) terminated with exit code None
[2024-12-27T17:40:40.894+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=510, status='terminated', started='16:35:40') (510) terminated with exit code None
[2024-12-27T17:40:40.898+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=8700, status='terminated', started='17:33:12') (8700) terminated with exit code None
[2024-12-27T17:40:41.024+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=475, status='terminated', started='16:35:38') (475) terminated with exit code None
[2024-12-27T17:40:41.041+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=476, status='terminated', started='16:35:38') (476) terminated with exit code None
[2024-12-27T17:40:41.057+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=449, status='terminated', started='16:35:38') (449) terminated with exit code None
[2024-12-27T17:40:40.800+0000] {taskinstance.py:3311} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/opt/src/extract.py", line 172, in movie_cast_filmmakers_extract
    driver.get(link)
  File "/home/airflow/.local/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 393, in get
    self.execute(Command.GET, {"url": url})
  File "/home/airflow/.local/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 382, in execute
    response = self.command_executor.execute(driver_command, params)
  File "/home/airflow/.local/lib/python3.9/site-packages/selenium/webdriver/remote/remote_connection.py", line 404, in execute
    return self._request(command_info[0], url, body=data)
  File "/home/airflow/.local/lib/python3.9/site-packages/selenium/webdriver/remote/remote_connection.py", line 428, in _request
    response = self._conn.request(method, url, body=body, headers=headers, timeout=self._client_config.timeout)
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/request.py", line 81, in request
    return self.request_encode_body(
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/request.py", line 173, in request_encode_body
    return self.urlopen(method, url, **extra_kw)
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/poolmanager.py", line 376, in urlopen
    response = conn.urlopen(method, u.request_uri, **kw)
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/connectionpool.py", line 716, in urlopen
    httplib_response = self._make_request(
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/connectionpool.py", line 468, in _make_request
    six.raise_from(e, None)
  File "<string>", line 3, in raise_from
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/connectionpool.py", line 463, in _make_request
    httplib_response = conn.getresponse()
  File "/usr/local/lib/python3.9/http/client.py", line 1377, in getresponse
    response.begin()
  File "/usr/local/lib/python3.9/http/client.py", line 320, in begin
    version, status, reason = self._read_status()
  File "/usr/local/lib/python3.9/http/client.py", line 281, in _read_status
    line = str(self.fp.readline(_MAXLINE + 1), "iso-8859-1")
  File "/usr/local/lib/python3.9/socket.py", line 716, in readinto
    return self._sock.recv_into(b)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 3095, in signal_handler
    raise AirflowTaskTerminated("Task received SIGTERM signal")
airflow.exceptions.AirflowTaskTerminated: Task received SIGTERM signal

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/connection.py", line 174, in _new_conn
    conn = connection.create_connection(
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/util/connection.py", line 95, in create_connection
    raise err
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/util/connection.py", line 85, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/connectionpool.py", line 716, in urlopen
    httplib_response = self._make_request(
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/connectionpool.py", line 416, in _make_request
    conn.request(method, url, **httplib_request_kw)
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/connection.py", line 244, in request
    super(HTTPConnection, self).request(method, url, body=body, headers=headers)
  File "/usr/local/lib/python3.9/http/client.py", line 1285, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "/usr/local/lib/python3.9/http/client.py", line 1331, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "/usr/local/lib/python3.9/http/client.py", line 1280, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "/usr/local/lib/python3.9/http/client.py", line 1040, in _send_output
    self.send(msg)
  File "/usr/local/lib/python3.9/http/client.py", line 980, in send
    self.connect()
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/connection.py", line 205, in connect
    conn = self._new_conn()
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/connection.py", line 186, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPConnection object at 0x7fb42c9be1c0>: Failed to establish a new connection: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 422, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/opt/src/extract.py", line 189, in movie_cast_filmmakers_extract
    driver.close()
  File "/home/airflow/.local/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 495, in close
    self.execute(Command.CLOSE)
  File "/home/airflow/.local/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 382, in execute
    response = self.command_executor.execute(driver_command, params)
  File "/home/airflow/.local/lib/python3.9/site-packages/selenium/webdriver/remote/remote_connection.py", line 404, in execute
    return self._request(command_info[0], url, body=data)
  File "/home/airflow/.local/lib/python3.9/site-packages/selenium/webdriver/remote/remote_connection.py", line 428, in _request
    response = self._conn.request(method, url, body=body, headers=headers, timeout=self._client_config.timeout)
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/request.py", line 77, in request
    return self.request_encode_url(
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/request.py", line 99, in request_encode_url
    return self.urlopen(method, url, **extra_kw)
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/poolmanager.py", line 376, in urlopen
    response = conn.urlopen(method, u.request_uri, **kw)
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/connectionpool.py", line 830, in urlopen
    return self.urlopen(
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/connectionpool.py", line 830, in urlopen
    return self.urlopen(
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/connectionpool.py", line 830, in urlopen
    return self.urlopen(
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/connectionpool.py", line 802, in urlopen
    retries = retries.increment(
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/util/retry.py", line 594, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=59133): Max retries exceeded with url: /session/9253bf0a7ea50665da88900a863fd7b2/window (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fb42c9be1c0>: Failed to establish a new connection: [Errno 111] Connection refused'))
[2024-12-27T17:40:41.558+0000] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=movie_pipeline, task_id=scrape_filmmakers, run_id=manual__2024-12-27T11:38:32.797674+00:00, execution_date=20241227T113832, start_date=20241227T163517, end_date=20241227T174041
[2024-12-27T17:40:41.742+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-27T17:40:41.749+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 78 for task scrape_filmmakers (HTTPConnectionPool(host='localhost', port=59133): Max retries exceeded with url: /session/9253bf0a7ea50665da88900a863fd7b2/window (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fb42c9be1c0>: Failed to establish a new connection: [Errno 111] Connection refused')); 267)
Traceback (most recent call last):
  File "/opt/src/extract.py", line 172, in movie_cast_filmmakers_extract
    driver.get(link)
  File "/home/airflow/.local/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 393, in get
    self.execute(Command.GET, {"url": url})
  File "/home/airflow/.local/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 382, in execute
    response = self.command_executor.execute(driver_command, params)
  File "/home/airflow/.local/lib/python3.9/site-packages/selenium/webdriver/remote/remote_connection.py", line 404, in execute
    return self._request(command_info[0], url, body=data)
  File "/home/airflow/.local/lib/python3.9/site-packages/selenium/webdriver/remote/remote_connection.py", line 428, in _request
    response = self._conn.request(method, url, body=body, headers=headers, timeout=self._client_config.timeout)
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/request.py", line 81, in request
    return self.request_encode_body(
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/request.py", line 173, in request_encode_body
    return self.urlopen(method, url, **extra_kw)
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/poolmanager.py", line 376, in urlopen
    response = conn.urlopen(method, u.request_uri, **kw)
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/connectionpool.py", line 716, in urlopen
    httplib_response = self._make_request(
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/connectionpool.py", line 468, in _make_request
    six.raise_from(e, None)
  File "<string>", line 3, in raise_from
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/connectionpool.py", line 463, in _make_request
    httplib_response = conn.getresponse()
  File "/usr/local/lib/python3.9/http/client.py", line 1377, in getresponse
    response.begin()
  File "/usr/local/lib/python3.9/http/client.py", line 320, in begin
    version, status, reason = self._read_status()
  File "/usr/local/lib/python3.9/http/client.py", line 281, in _read_status
    line = str(self.fp.readline(_MAXLINE + 1), "iso-8859-1")
  File "/usr/local/lib/python3.9/socket.py", line 716, in readinto
    return self._sock.recv_into(b)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 3095, in signal_handler
    raise AirflowTaskTerminated("Task received SIGTERM signal")
airflow.exceptions.AirflowTaskTerminated: Task received SIGTERM signal

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/connection.py", line 174, in _new_conn
    conn = connection.create_connection(
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/util/connection.py", line 95, in create_connection
    raise err
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/util/connection.py", line 85, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/connectionpool.py", line 716, in urlopen
    httplib_response = self._make_request(
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/connectionpool.py", line 416, in _make_request
    conn.request(method, url, **httplib_request_kw)
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/connection.py", line 244, in request
    super(HTTPConnection, self).request(method, url, body=body, headers=headers)
  File "/usr/local/lib/python3.9/http/client.py", line 1285, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "/usr/local/lib/python3.9/http/client.py", line 1331, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "/usr/local/lib/python3.9/http/client.py", line 1280, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "/usr/local/lib/python3.9/http/client.py", line 1040, in _send_output
    self.send(msg)
  File "/usr/local/lib/python3.9/http/client.py", line 980, in send
    self.connect()
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/connection.py", line 205, in connect
    conn = self._new_conn()
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/connection.py", line 186, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPConnection object at 0x7fb42c9be1c0>: Failed to establish a new connection: [Errno 111] Connection refused

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 3005, in _run_raw_task
    return _run_raw_task(
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 3159, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 3183, in _execute_task
    return _execute_task(self, context, task_orig)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 422, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/opt/src/extract.py", line 189, in movie_cast_filmmakers_extract
    driver.close()
  File "/home/airflow/.local/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 495, in close
    self.execute(Command.CLOSE)
  File "/home/airflow/.local/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 382, in execute
    response = self.command_executor.execute(driver_command, params)
  File "/home/airflow/.local/lib/python3.9/site-packages/selenium/webdriver/remote/remote_connection.py", line 404, in execute
    return self._request(command_info[0], url, body=data)
  File "/home/airflow/.local/lib/python3.9/site-packages/selenium/webdriver/remote/remote_connection.py", line 428, in _request
    response = self._conn.request(method, url, body=body, headers=headers, timeout=self._client_config.timeout)
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/request.py", line 77, in request
    return self.request_encode_url(
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/request.py", line 99, in request_encode_url
    return self.urlopen(method, url, **extra_kw)
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/poolmanager.py", line 376, in urlopen
    response = conn.urlopen(method, u.request_uri, **kw)
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/connectionpool.py", line 830, in urlopen
    return self.urlopen(
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/connectionpool.py", line 830, in urlopen
    return self.urlopen(
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/connectionpool.py", line 830, in urlopen
    return self.urlopen(
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/connectionpool.py", line 802, in urlopen
    retries = retries.increment(
  File "/home/airflow/.local/lib/python3.9/site-packages/urllib3/util/retry.py", line 594, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=59133): Max retries exceeded with url: /session/9253bf0a7ea50665da88900a863fd7b2/window (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fb42c9be1c0>: Failed to establish a new connection: [Errno 111] Connection refused'))
[2024-12-27T17:40:42.052+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=267, status='terminated', exitcode=1, started='16:35:17') (267) terminated with exit code 1
[2024-12-27T17:40:42.055+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=427, status='terminated', started='16:35:38') (427) terminated with exit code None
